pkg/util/validation/limits.go

```diff
@@ -64,10 +64,11 @@ type Limits struct {
 	MaxGlobalSeriesPerUser   int `yaml:"max_global_series_per_user" json:"max_global_series_per_user"`
 	MaxGlobalSeriesPerMetric int `yaml:"max_global_series_per_metric" json:"max_global_series_per_metric"`
 	// Metadata
-	MaxLocalMetricsWithMetadataPerUser  int `yaml:"max_metadata_per_user" json:"max_metadata_per_user"`
-	MaxLocalMetadataPerMetric           int `yaml:"max_metadata_per_metric" json:"max_metadata_per_metric"`
-	MaxGlobalMetricsWithMetadataPerUser int `yaml:"max_global_metadata_per_user" json:"max_global_metadata_per_user"`
-	MaxGlobalMetadataPerMetric          int `yaml:"max_global_metadata_per_metric" json:"max_global_metadata_per_metric"`
+	MaxLocalMetricsWithMetadataPerUser  int            `yaml:"max_metadata_per_user" json:"max_metadata_per_user"`
+	MaxLocalMetadataPerMetric           int            `yaml:"max_metadata_per_metric" json:"max_metadata_per_metric"`
+	MaxGlobalMetricsWithMetadataPerUser int            `yaml:"max_global_metadata_per_user" json:"max_global_metadata_per_user"`
+	MaxGlobalMetadataPerMetric          int            `yaml:"max_global_metadata_per_metric" json:"max_global_metadata_per_metric"`
+	OutOfOrderTimeWindow                model.Duration `yaml:"out_of_order_time_window" json:"out_of_order_time_window"`
```

Comment:
it's not related to metadata, so this is probably not the right place

Reply:
Good catch. I just wanted to add it to the ingester configs here because we don't have OOO specific configs anyway.
Now I added a comment about the OOO section, hope this is clear now.